mqtt/mqttd/src/broker/mod.rs

```diff
@@ -35,6 +37,9 @@ where
     let (mut snapshotter_shutdown_handle, snapshotter_join_handle) =
         start_snapshotter(broker.handle(), persistor).await;
 
+    info!("starting bridge...");
+    start_bridge().await?;
```

Comment:
I'm thinking we will need to shutdown the bridge similar to how we shutdown the command handler. There is a method in the bootstrap/edgehub.rs called ```start_sidecars()``` where you can move this start / stop logic.